x-pack/plugins/cross_cluster_replication/public/app/constants/form_schemas.js

```diff
@@ -0,0 +1,137 @@
+      }),
+      validator: Joi.number().allow(''),
+    },
+    maxWriteRequestSize: {
+      label: i18n.translate('xpack.crossClusterReplication.followerIndexForm.advancedSettings.maxWriteRequestSizeTitle', {
+        defaultMessage: 'Max write request size'
+      }),
+      description: i18n.translate('xpack.crossClusterReplication.followerIndexForm.advancedSettings.maxWriteRequestSizeDescription', {
+        defaultMessage: 'The maximum total bytes of operations per bulk write request executed on the follower.'
+      }),
+      validator: Joi.number().allow(''),
+    },
+    maxOutstandingWriteRequests: {
+      label: i18n.translate('xpack.crossClusterReplication.followerIndexForm.advancedSettings.maxOutstandingWriteRequestsTitle', {
+        defaultMessage: 'Max outstanding write requests'
+      }),
+      description: i18n.translate('xpack.crossClusterReplication.followerIndexForm.advancedSettings.maxOutstandingWriteRequestsDescription', {
+        defaultMessage: 'The maximum number of outstanding write requests on the follower.'
+      }),
+      validator: Joi.number().allow(''),
+    },
+    maxWriteBufferCount: {
+      label: i18n.translate('xpack.crossClusterReplication.followerIndexForm.advancedSettings.maxWriteBufferCountTitle', {
+        defaultMessage: 'Max write buffer count'
+      }),
+      description: i18n.translate('xpack.crossClusterReplication.followerIndexForm.advancedSettings.maxWriteBufferCountDescription', {
+        defaultMessage: 'The maximum number of operations that can be queued for writing; when this limit is reached, reads from the remote cluster will be deferred until the number of queued operations goes below the limit.'
+      }),
+      validator: Joi.number().allow(''),
+    },
+    maxWriteBufferSize: {
+      label: i18n.translate('xpack.crossClusterReplication.followerIndexForm.advancedSettings.maxWriteBufferSizeTitle', {
+        defaultMessage: 'Max write buffer size'
+      }),
+      description: i18n.translate('xpack.crossClusterReplication.followerIndexForm.advancedSettings.maxWriteBufferSizeDescription', {
+        defaultMessage: 'The maximum total bytes of operations that can be queued for writing; when this limit is reached, reads from the remote cluster will be deferred until the total bytes of queued operations goes below the limit.'
+      }),
+      validator: Joi.number().allow(''),
+    },
+    maxRetryDelay: {
+      label: i18n.translate('xpack.crossClusterReplication.followerIndexForm.advancedSettings.maxRetryDelayTitle', {
+        defaultMessage: 'Max retry delay'
+      }),
+      description: i18n.translate('xpack.crossClusterReplication.followerIndexForm.advancedSettings.maxRetryDelayDescription', {
+        defaultMessage: 'The maximum time to wait before retrying an operation that failed exceptionally; an exponential backoff strategy is employed when retrying.'
+      }),
+      validator: Joi.number().allow(''),
```

Comment:
this is a time unit value, so should be validated as a string:
https://www.elastic.co/guide/en/elasticsearch/reference/master/common-options.html#time-units